manifests/g/Gyan/CodexFFmpeg/5.1.2/Gyan.CodexFFmpeg.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-2-7.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Gyan.CodexFFmpeg
+PackageVersion: 5.1.2
+InstallerType: zip
+ReleaseDate: 2022-09-26
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\bin\ffmpeg.exe
+    PortableCommandAlias: ffmpeg
+  - RelativeFilePath: .\bin\ffplay.exe
+    PortableCommandAlias: ffplay
+  - RelativeFilePath: .\bin\ffprobe.exe
+    PortableCommandAlias: ffprobe
```

Fix `RelativeFilePath`:
```suggestion
  - RelativeFilePath: .\ffmpeg-5.1.2-full_build\bin\ffmpeg.exe
    PortableCommandAlias: ffmpeg
  - RelativeFilePath: .\ffmpeg-5.1.2-full_build\bin\ffplay.exe
    PortableCommandAlias: ffplay
  - RelativeFilePath: .\ffmpeg-5.1.2-full_build\bin\ffprobe.exe
    PortableCommandAlias: ffprobe
```